plugins/opensource/fullscreen.md

```diff
@@ -27,7 +27,6 @@ tinymce.init({
   toolbar: 'fullscreen'
 });
 ```
-
```

Comment:
Looks like this line was accidentally removed between all the changes?

Reply:
To quote my commit message:

> A blank line was accidentally removed when undoing an edit. The blank line has been restored.